split up ecsql reference into indiviual files, remove iframed iModel … (#6377)
iTwin/itwinjs-core: docs/learning/ECSqlReference/ECSqlOptions.md

# ECSQLOPTIONS or OPTIONS clause

> `ECSQLOPTIONS` which can also be written as just `OPTIONS` use to specify flags thats will effect processing of ECSQL statement.

Reviewer: ```suggestion
`ECSQLOPTIONS` (or `OPTIONS` for short) are used to specify flags thats will effect processing of ECSQL statements.
```

Author: ```suggestion
`ECSQLOPTIONS` (or `OPTIONS` for short) are used to specify flags thats will effect processing of the ECSQL statement.
```
